docs/user-guide/service-accounts.md

```diff
@@ -120,14 +120,41 @@ $ kubectl delete serviceaccount/build-robot
 ```
 
 <!-- TODO: describe how to create a pod with no Service Account. -->
+Note that if a pod does not have a `ServiceAccount` set, the `ServiceAccount` will be set to `default`.
 
 ## Adding Secrets to a service account.
```

Comment:
The doc you've added here should be in a section named something like "Manually create a service account API token"

The TODO about "Test and explain how to use additional non-K8s secrets with an existing service account." still applies, and should describe how to do things like create ssh private key secrets, or docker image pull secrets, and add them to a service account. That can be part of a different PR, but don't remove the TODO yet